dbms/src/TestUtils/ExecutorTestUtils.h

```diff
@@ -52,6 +50,28 @@ class ExecutorTest : public ::testing::Test
 
     void executeInterpreter(const String & expected_string, const std::shared_ptr<tipb::DAGRequest> & request, size_t concurrency);
 
+    enum SourceType
+    {
+        TableScan,
+        ExchangeReceiver
+    };
+
+    // for single source query, the source executor name is ${type}_0
+    static String getSourceName(SourceType type)
+    {
+        switch (type)
+        {
+        case TableScan:
+            return "table_scan_0";
+        case ExchangeReceiver:
+            return "exchange_receiver_0";
+        default:
+            throw Exception(ErrorCodes::BAD_ARGUMENTS,
```

Comment:
In those cases, do we really need to throw an exception? maybe just ignoring is enough?